images/Dockerfile.centos

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.centos.org/centos:8 AS build
+FROM docker.io/library/almalinux:8 AS build
```

Comment:
I think it should be `docker.io/almalinux`

Reply:
thanks